### PR TITLE
 create actor fragment

### PR DESCRIPTION
**Pull request by meoconlontonbebong**
Link  redmine : https://edu-redmine.framgia.vn/issues/6824
 Create Actor Info Fragment .

### CODE REVIEW

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/res/layout/item_actor_info.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                android:padding="@dimen/dp_16"
+                android:src="@drawable/ic_play_circle"
+                app:bindImage="@{viewModel.actorObservable.profilePath}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="19dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_actor_info.xml]: hardcode

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/res/layout/fragment_actor.xml [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+            type="com.example.moviedb_37.screen.moviedetails.MovieDetailsViewModel" />
+    </data>
+
+    <android.support.constraint.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/recycler_actor"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_actor.xml]: wrong name

**Comment by meoconlontonbebong** [comment]
ready

File: app/src/main/res/layout/fragment_actor.xml (hunk 2 above)
Comment (meoconlontonbebong) [reply app/src/main/res/layout/fragment_actor.xml]: Anh muốn thêm s phía sau hay ntn ạ , hay đổi actors thành cast ạ ,

**Pull request merged**